sprints/dashboard/models.py

```diff
@@ -84,9 +84,9 @@ def __init__(
             # Possible for epics
             self.current_sprint = False
         try:
-            self.story_points = int(getattr(issue.fields, issue_fields['Story Points'], 0))
+            self.story_points = getattr(issue.fields, issue_fields['Story Points'], None)
         except (AttributeError, TypeError):
```

Comment:
I suppose we don't need this exception anymore, as we're not casting the value?

Reply:
Yep